devenv/dev-dashboards/migrations/migrations.json

```diff
@@ -194,7 +218,7 @@
         "sort": 0,
         "value_type": "individual"
       },
-      "type": "timeseries",
+      "type": "graph",
```

Comment:
not sure why this was set to timeseries when purpose of the gdev dashboard is to test auto migrations of deprecated panels 🤷‍♂️

Reply:
yeah, odd